src/Components/ProvisioningWizard/steps/index.js

```diff
@@ -32,7 +42,7 @@ const defaultSteps = ({
             provider={provider}
             architecture={architecture || 'x86_64'}
             composeID={id}
-            imageSourceID={sourceId}
+            imageSourceID={sourceIDs?.[0]}
```

Comment:
This is temporary, to keep this small, follow up should be to get the Sources select to filter Sources the same way.